### DISCUSSION TITLE
IS it possible to get a CI Token that is valid for more than 30 days?

### DISCUSSION THREAD

**Discussion by eiswind** [onepub-dev/onepub#546] [Support] [answered]
I'm just getting started, but I feel the perspective of updating the tokens every 30 days in all of my CI Pipelines seems very annoying. Can you think of an alternative for CI purposes?

**Comment by bsutton** [comment, marked as answer]
The 30 days is renewed each time you use the token.

I will check the doco and make sure it reflects this.

**Comment by bsutton** [comment]
The FAQ has been updated.

Apologies for the confusion.

Happy to set up a zoom if you want some help getting started.

**Reply by eiswind** [reply]
Thanks for the offer. So far everything works like a charm!

**Comment by bsutton** [comment]
That's what I like to hear:) If anything changes feel free to reach out.
